ow_faults/src/FaultInjector.cpp

```diff
@@ -85,118 +98,122 @@ void FaultInjector::jointStateCb(const sensor_msgs::JointStateConstPtr& msg)
   }
 
   sensor_msgs::JointState output(*msg);
+  std_msgs::Float64 soc_msg;
+
```

Comment:
why did you need to move this up?

Reply:
just so that it would live with all the other declarations. It felt more random in the middle.

Reply:
Well in C++ it is best if you delay the declaration of stack variables till the point you are going to use it.
In any case, this really hints that this method is quite long .. consider breaking it into smaller parts later on (i.e when you start another related-PR)